### PR TITLE
feature/taxonomy colors

### PR DESCRIPTION
**Pull request by martinyde**
https://jira.itkdev.dk/browse/LOOP-883

- LOOP-883: Added templates and styling for taxonomy colors
- LOOP-883: Added config for color picker

### CODE REVIEW

File: web/profiles/custom/os2loop/themes/os2loop_theme/templates/content-entities/paragraph--os2loop-documents-highlighted-co.html.twig [hunk 1]
```diff
@@ -43,7 +43,8 @@
     'os2loop-highlighted-container',
     'paragraph--type--' ~ paragraph.bundle|clean_class,
     view_mode ? 'paragraph--view-mode--' ~ view_mode|clean_class,
-    not paragraph.isPublished() ? 'paragraph--unpublished'
+    not paragraph.isPublished() ? 'paragraph--unpublished',
+    drupal_field('os2loop_shared_subject', 'node', paragraph.parent_id.value)['0']['#options'].entity.os2loop_taxonomy_color_picker.value
```
Comment (rimi-itk) [inline web/profiles/custom/os2loop/themes/os2loop_theme/templates/content-entities/paragraph--os2loop-documents-highlighted-co.html.twig:47]: We should add a check for if this value (and indexes etc.) actually is defined; probably just appending `|default('')` will work.

File: web/profiles/custom/os2loop/themes/os2loop_theme/templates/field/field--os2loop-shared-subject.html.twig [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    {% endfor %}
+  {% endif %}
+{% elseif label_display == 'inline' %}
+  <span{{ attributes }}>
+    <span{{ title_attributes.addClass(title_classes) }}>{{ label }}:</span>
+    {% if multiple %}
+      <span>
+    {% endif %}
+    {% for item in items %}
+      <span{{ item.attributes.addClass(item.content['#options'].entity.os2loop_taxonomy_color_picker.value, 'shared-subject') }}>{{ item.content }}</span>
```
Comment (rimi-itk) [inline web/profiles/custom/os2loop/themes/os2loop_theme/templates/field/field--os2loop-shared-subject.html.twig:64]: Check that this value actually exists and consider moving it to a variable to improve readability.

**Review by rimi-itk — CHANGES_REQUESTED** [review]
Minor changes and suggestions.

File: config/sync/core.entity_view_display.taxonomy_term.os2loop_subject.default.yml [hunk 3]
```diff
@@ -14,6 +17,13 @@ content:
     region: content
     settings: {  }
     third_party_settings: {  }
+  os2loop_taxonomy_color_picker:
+    weight: 1
+    label: above
+    settings: {  }
+    third_party_settings: {  }
+    type: list_default
+    region: content
```
[the remark this answers is not in the document]
Comment (martinyde) [reply config/sync/core.entity_view_display.taxonomy_term.os2loop_subject.default.yml]: Removed

File: web/profiles/custom/os2loop/themes/os2loop_theme/templates/content-entities/paragraph--os2loop-documents-highlighted-co.html.twig (hunk 1 above)
Comment (martinyde) [reply web/profiles/custom/os2loop/themes/os2loop_theme/templates/content-entities/paragraph--os2loop-documents-highlighted-co.html.twig]: Added |default('')

File: web/profiles/custom/os2loop/themes/os2loop_theme/templates/field/field--os2loop-shared-subject.html.twig (hunk 2 above)
Comment (martinyde) [reply web/profiles/custom/os2loop/themes/os2loop_theme/templates/field/field--os2loop-shared-subject.html.twig]: Set variable

File: web/profiles/custom/os2loop/themes/os2loop_theme/templates/field/field--os2loop-shared-subject.html.twig [hunk 4]
```diff
@@ -61,7 +61,8 @@
       <span>
     {% endif %}
     {% for item in items %}
-      <span{{ item.attributes.addClass(item.content['#options'].entity.os2loop_taxonomy_color_picker.value, 'shared-subject') }}>{{ item.content }}</span>
+      {% set color_picker_value = item.content['#options'].entity.os2loop_taxonomy_color_picker.value %}
```
Comment (rimi-itk) [inline web/profiles/custom/os2loop/themes/os2loop_theme/templates/field/field--os2loop-shared-subject.html.twig:64]: Add `|default('')` to be sure that this does not fail.

**Review by rimi-itk — CHANGES_REQUESTED** [review]

**Review by rimi-itk — APPROVED** [review]

**Pull request merged**